docker/substra-backend/Dockerfile

```
@@ -11,11 +11,11 @@ COPY ./backend/requirements.txt /usr/src/app/.
 
```

```suggestion
RUN apt-get update \
    && apt-get install -y --no-install-recommends \
        python3 \
        python3-pip  \
        python3-dev \
        build-essential \
        libssl-dev \
        libffi-dev \
       libxml2-dev \
       libxslt1-dev \
       zlib1g-dev \
       git \
       curl \
       netcat \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*

```
could be done in one layer + reduce layer size by cleaning noisy files